docker/jupyter/scripts/extension/jupyter_install_lab_extension.sh

```diff
@@ -5,6 +5,17 @@ set -e
 
 if [ -n "$1" ]; then
   JUPYTER_EXTENSION=$1
-  jupyter labextension install $JUPYTER_EXTENSION
+  if [[ ${JUPYTER_EXTENSION} == *'.js' ]]; then
+    # use jupyterlab extension template to create an extension using the specified JS file
+    # see https://github.com/jupyterlab/extension-cookiecutter-js
+    JUPYTER_EXTENSION_NAME=`basename ${JUPYTER_EXTENSION%%.*}`
+    cookiecutter --no-input https://github.com/jupyterlab/extension-cookiecutter-js extension_name=${JUPYTER_EXTENSION_NAME}
+    cp -f ${JUPYTER_EXTENSION} ${JUPYTER_EXTENSION_NAME}/lib/plugin.js
+    cd ${JUPYTER_EXTENSION_NAME}
+    jlpm
```

Comment:
In this [tutorial](https://jupyterlab.readthedocs.io/en/stable/developer/xkcd_extension_tutorial.html) it's:
```
cd $ext
jlpm install
jupyter labextension install . --no-build
```
Is that equivalent? (I _think_ so?)

Reply:
I just didn't want to specify --no-build because the user would have to reload the JupyterLab page once the cluster is ready. Other than that it should be equivalent!